app/src/main/java/co/minesweepers/birthday/adapters/helpers/ItemTouchHelperCallback.java

```diff
@@ -0,0 +1,35 @@
+package co.minesweepers.birthday.adapters.helpers;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.helper.ItemTouchHelper;
+
+public class ItemTouchHelperCallback extends ItemTouchHelper.Callback {
+
+    private ItemTouchEventsListener mListener;
+
+    public ItemTouchHelperCallback(@NonNull ItemTouchEventsListener listener) {
+        mListener = listener;
+    }
+
+    @Override
+    public int getMovementFlags(RecyclerView recyclerView, RecyclerView.ViewHolder viewHolder) {
+        int dragFlags = ItemTouchHelper.UP | ItemTouchHelper.DOWN;
+        return makeMovementFlags(dragFlags, 0); // TODO: allow swipe movement for deleting
```

Comment:
swipe to delete seems very iOS like action .  In most places on android, standard seems to be long press and then a delete button shows up . Lets check for a few examples before deciding to implement that one .